src/globus_cli/termio/formatters/primitive.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+import datetime
+import typing as t
+
+from .base import FieldFormatter
+
+
+class StrFormatter(FieldFormatter[str]):
+    def parse(self, value: t.Any) -> str:
+        return str(value)
+
+    def render(self, value: str) -> str:
+        return value
+
+
+class DateFormatter(FieldFormatter[datetime.datetime]):
+    def parse(self, value: t.Any) -> datetime.datetime:
+        return datetime.datetime.fromisoformat(value)
```

Comment:
No, you didn't miss anything. I'll need to add the relevant guard (and it would be nice to add a test case to exercise this too, which I'll try to do)